src/main/java/seedu/savenus/model/Menu.java

```diff
@@ -3,13 +3,10 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 import javafx.collections.ObservableList;
 import seedu.savenus.model.food.Food;
 import seedu.savenus.model.food.UniqueFoodList;
-import seedu.savenus.model.purchase.Purchase;
-import seedu.savenus.model.purchase.PurchaseHistory;
 import seedu.savenus.model.wallet.Wallet;
```

Comment:
Great that you've removed PurchaseHistory :) 

I think for next week you just need to focus on pulling wallet out of the menu to show great Object Oriented Programming! 

Not a crucial work for now.